tach.toml

```diff
@@ -39,15 +25,21 @@ depends_on = [
     "ee",
     "common.hogql_parser",
     "common.hogvm.python",
+
+    # NOTE: Add new product dependencies here and in settings/web.py PRODUCTS_APPS
     "products.early_access_features",
+    "products.editor",
+    "products.revenue_analytics",
```

Comment:
Just a note to check if the style changes are due to a local or different formatter; nothing is required on this PR.